assets/js/translator.js

```diff
@@ -730,7 +731,7 @@ function detectLanguage() {
 }
 
 function translationNotAvailable() {
-    $('#translatedText').text(dynamicLocalizations['Not_Available']);
+    $('#translatedText').val(dynamicLocalizations['Not_Available']);
```

Comment:
Can we actually just change both val and text? I'm not sure of the intricacies behind this and it won't hurt.